contracts/interchain-token-service/src/executable.rs

```diff
@@ -7,14 +7,42 @@
 
 use soroban_sdk::{contractclient, Address, Bytes, BytesN, Env, String};
```

Comment:
Re-export `Executable` macro from this crate? Feels a bit weird for apps to import ITS specific macro from `stellar-axelar-std`